src/devices/Ads1115/DataRate.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    ///  Control the Data Rate 
+    /// </summary>
+    public enum DataRate
+    {
+        SPS008 = 0x00,
```

Comment:
NIT: Add comments above each one of this specifying what does each data rate really translates as.